src/c_api/c_api_symbolic.cc

```diff
@@ -708,6 +787,31 @@ int MXSymbolInferShapePartialEx(SymbolHandle sym,
                               &succ);
 }
 
+int MXSymbolInferShapePartialExInt64(SymbolHandle sym,
```

Comment:
If the internal implementation is the same and only differs by data type, we can create an internal template to do this?